spec/dotloop/loop_spec.rb

```diff
@@ -37,5 +37,61 @@
         ).attributes
       )
     end
+
+    it 'accepts parameter status_ids' do
+      expect(client).to receive(:get).with(
+        '/profile/1234/loop',
+        batchNumber: 1, batchSize: 50, statusIds: [1, 2, 3]
+      ).and_return([])
+      subject.all(profile_id: '1234', options: { status_ids: [1, 2, 3] })
```

Comment:
I'd prefer to read this as 

```
subject.all(profile_id: '1234', status_ids: [1, 2, 3])
```

Because that's how we'll interact with it (ideally...it should *just work*)